salt/journal-cms/config/etc-nginx-sites-enabled-journal-cms.conf

```diff
@@ -16,7 +16,6 @@ server {
 
     location = /favicon.ico {
         log_not_found off;
-        access_log off;
```

Comment:
this is applying to journal-cms, not to the api-dummy; sure you want to change this?